### ISSUE TITLE
Generalize participants.tsv reference

### ISSUE THREAD

**Issue by arokem** [nrdg/hbn_viewer#36]
This code currently refers to the first two columns of a tsv file:

https://github.com/nrdg/hbn_viewer/blob/27e3c2ca46ce67712b6ff3ee4c9f2c12a6a66a74/src/App.vue#L54-L78

But maybe we want to detect which is the subject id and which is the session id instead.

**Comment by IsaacCrane** [comment]
The tsv file is a and newArray is the array that is being built where each element can be generalized as [subject_id, session]. I think the only assumption is that the tags are participant_id and site. i just says the row in the tsv file that is being referred to. I don't think this code ever refers to columns in the tsv file. What the above code does to make the tsv file iterable is turn it to an array where each element is a class with the labels at the top being the attributes. We are making newArray as we go and then checking to see if a subject is already in there in case there are multiple sessions. This could be the wrong way of doing that but I think the only times that we are actually referring to columns is in the newArray which we are specifically writing to have that column structure. 

TLDR: I think this is right. The difference between newArray and a is confusing because they are named poorly.

**Comment by arokem** [comment]
Thanks! It sounds like this will be refactored in code that @asagilmore is currently working on.
